modelci/hub/deployer/pytorch/deploy_model_cpu.sh

```diff
@@ -2,4 +2,4 @@
 
 docker run -d --rm -p "${2}":8000 -p "${3}":8001 \
   --mount type=bind,source="${HOME}"/.modelci/"${1}"/pytorch-torchscript,target=/models/"${1}" \
-  -e MODEL_NAME="${1}" -t mlmodelci/pytorch-serving:latest
+  -e MODEL_NAME="${1}" --env-file docker-env.env -t mlmodelci/pytorch-serving:latest
```

Comment:
in the CPU version, docker-env is not needed. `docker-env.env` is to specify GPU device